rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1370,20 +1371,16 @@ static void * init_and_alloc_sample(
       return nullptr);
   }
   // allocate memory for message + header
-  auto chunk_ptr = dds_data_allocator_alloc(
-    &entity->data_allocator,
-    DETERMINE_ICEORYX_CHUNK_SIZE(sample_size));
+  // the header will be initialized and the chunk pointer will be returned
```

Comment:
We need this because we have no publisher in general to allocate/loan from, right?

Reply:
That is correct, we don't have access to the publisher to loan from